docs/CONTRIBUTING.md

```diff
@@ -669,126 +669,82 @@ Hello, World!
+the engine with its embedded language server. To build the engine issue
+following command in the _sbt prompt_:
 
 ```bash
-cat ~/enso/projects/Unnamed/package.yaml
+sbt:enso> buildEngineDistribution
 ```
 
-```yaml
-name: Unnamed
-namespace: local
-version: 0.0.1
-license: ""
-authors: []
-maintainers: []
-edition: "2021.20-SNAPSHOT"
-prefer-local-libraries: true
-```
+Once all the components are assembled, it is time to execute them in
+orchestration. One can pass following environment variables to
+`project-manager`:
 
-We need to set `edition` to a value that will represent the development version.
-It should be different from any Enso versions that have already been released.
-In this case, we chose the `2021.20-SNAPSHOT` (the current development edition).
-The project manager will look for the appropriate subdirectory in the _engines_
-directory of the distribution folder. Distribution paths are printed when you
-run project manager with `-v` verbose logging.
+- `ENSO_JVM_OPTS` to for example turn
+  [debugging of the Engine runtime](debugger/README.md) on
+- `ENSO_JVM_PATH` to force a fixed GraalVM to execute the engine/language server
+  process on
+- `ENSO_ENGINE_PATH` the path to engine/language server as created by
+  `buildEngineDistribution`
 
-Btw. you can specify `ENSO_JVM_OPTS` to turn
-[debugging of the Engine runtime](debugger/README.md) on:
+One doesn't need to deal with these options directly, there is an _sbt command_
+to orchestrate them all:
 
 ```bash
-$ export ENSO_JVM_OPTS=-agentlib:jdwp=transport=dt_socket,address=5005
-$ ./built-distribution/enso-project-manager-0.0.0-dev-linux-amd64/enso/bin/project-manager --no-log-masking -v
-[info] [2021-06-16T11:49:33.639Z] [org.enso.projectmanager.boot.ProjectManager$] Starting Project Manager...
-[debug] [2021-06-16T11:49:33.639Z] [org.enso.runtimeversionmanager.distribution.DistributionManager] Detected paths: DistributionPaths(
-  dataRoot = /home/dbv/.local/share/enso,
-  runtimes = /home/dbv/.local/share/enso/runtime,
-  engines  = /home/dbv/.local/share/enso/dist,
-  bundle   = None,
-  config   = /home/dbv/.config/enso,
-  locks    = /run/user/1000/enso/lock,
-  tmp      = /home/dbv/.local/share/enso/tmp
-)
+sbt:enso> runProjectManagerDistribution
 ```
 
-On Linux it looks for the `~/.local/share/enso/dist/0.2.32-SNAPSHOT/` directory.
-
-We can build an engine distribution using the `buildEngineDistribution` command
-in SBT.
+The above command invokes `buildProjectManagerDistribution`,
+`buildEngineDistribution` and then uses `ENSO_ENGINE_PATH` to connect them
+together and also specifies the `ENSO_JVM_PATH` to the JVM `sbt` process runs
+on.
 
-##### Bash
+There also is a simple way to [debug](debugger/README.md). When adding `--debug`
+option to the _sbt command_:
 
 ```bash
-sbt buildEngineDistribution
-```
-
-##### PowerShell
-
-```powershell
-sbt.bat buildEngineDistribution
+sbt:enso> runProjectManagerDistribution --debug
```

Comment:
This `--debug` actually debugs the engine. The instructions how to debug the project manager itself are given at
- #7153

An alternative would be to require an argument to the `--debug` option. Either `engine` or `project-manager` and based on that debug the right process in the system.

Btw. to support debugging `project-manager` we would need to debug the JVM version (not the _native image_ version).